nav2_bringup/bringup/launch/spawn_robot_launch.py

```diff
@@ -13,22 +13,24 @@
 # limitations under the License.
 
 import os
-import sys
+
 from launch import LaunchDescription
-from launch.substitutions import EnvironmentVariable
+
 import launch.actions
 import launch_ros.actions
 
 def generate_launch_description():
-    world = launch.substitutions.LaunchConfiguration('world')
 
     return LaunchDescription([
-        launch.actions.DeclareLaunchArgument(
-            'world', description='Full path to world file to load'),
-
-        launch.actions.ExecuteProcess(
-            cmd=['gazebo', '--verbose', '-s', 'libgazebo_ros_init.so', world],
-            output='screen'),
-
-        launch_ros.actions.Node(package='rviz2', node_executable='rviz2', output='screen')
+        #TODO(orduno) might not be necessary to have it's own package
+        launch_ros.actions.Node(
+            package='spawn_turtlebot3',
+            node_executable='spawn_turtlebot',
+            output='screen',
+            arguments=[
+                    launch.substitutions.LaunchConfiguration('robot_name'),
+                    launch.substitutions.LaunchConfiguration('robot_name'),
```

Comment:
Duplicate line.

Reply:
It's actually two different inputs, the name for the entity and the namespace for it. In this case, we set both the same.